README.md

```diff
@@ -1,280 +1,261 @@
+peers. Additionally, you might want to mount some volumes to persist configuration and chain data. For instance, to run
+a non-validator node:
 
 ```sh
-...
-$ docker run -ti -v $(pwd)/database:/data -v $(pwd)/config:/config/custom -p 30300:30300 trustlines/tlbc-testnet --role validator --address MY_ADDRESS
+$ mkdir -p trustlines/data-laika trustlines/config trustlines/enode trustlines/shared
+$ docker run -d --name laika-node --network network-laika \
+    -v $(pwd)/trustlines/data-laika:/data \
+    -v $(pwd)/trustlines/config:/config/custom \
+    -v $(pwd)/trustlines/enode:/config/network \
+    -v $(pwd)/trustlines/shared:/shared/ \
+    -p 30300:30300 -p 30300:30300/udp \
+    trustlines/tlbc-testnet
 ```
 
-#### Create New Account
-
-If no already existing account is available or a new one should be created anyway, _Parity_ could be used to do so. Please consider other
-options like [MetaMask](https://metamask.io/), [Mist](https://github.com/ethereum/mist) or any other (online) wallet tool.
-In relation to the instructions for the [participant](#participant) and [validator](#validator) roles, we use the folder called `config` to
-bind as _Docker_ volume to _Parity_. Afterwards the key will be placed there and the first steps of these instructions can be skipped.
+If you are a validator, this sequence of commands will supply Parity with your keystore file, password, and address so
+that it can produce blocks:
 
 ```sh
-$ mkdir ./config
-$ docker run -ti -v $(pwd)/config/:/config/custom trustlines/tlbc-testnet --parity-args account new
+$ mkdir -p trustlines/data-laika trustlines/config/keys/Trustlines trustlines/enode trustlines/shared
+$ cp /path/to/your/keystore/file.json trustlines/config/keys/Trustlines
+$ echo "<passphrase_for_keystore_file>" > trustlines/config/pass.pwd
+$ docker run -d --name laika-node --network network-laika \
+    -v $(pwd)/trustlines/data-laika:/data \
+    -v $(pwd)/trustlines/config:/config/custom \
+    -v $(pwd)/trustlines/enode:/config/network \
+    -v $(pwd)/trustlines/shared:/shared/ \
+    -p 30300:30300 -p 30300:30300/udp \
+    trustlines/tlbc-testnet
 ```
 
-_Parity_ will ask for a password, that should be stored by you to `./config/pass.pwd` afterwards. The address corresponding to the generated
-private key gets printed out on the commandline at the last line starting with `0x`. Please copy it for the later use. It will be needed for
-the `--address` argument where it will be added in plain text.
-
----
-
-### Without Docker
+#### Netstats Client
 
-This section explains how to start a local peer without using the _Docker_ image.
+The netstats client reports the state of your node to the
+[Laika netstats page](https://laikanetstats.trustlines.foundation/) that gives a rough overview of the current network state.
+It is a fully optional component which helps the community, at the cost leaks of some leakage of
+private information.
```

Comment:
In correct English: `It is a fully optional component which helps the community at the cost of leakage of some private information.`
I would simplify it as: `It is an optional component which helps the community by providing information on your running node to a central server.`